src/tizen-client/src/src/components/Movie/Card/MovieCard.tsx

```diff
@@ -7,13 +7,15 @@ import "./MovieCard.styl";
 
 interface Props {
   movie: Movie;
+
+  clickAction: Function;
```

Comment:
@justandreyb please use on*Event* notation, e.g. onClick

Reply:
@justandreyb Also do not use Function as type of parameter. `() => void` is your choice